numpy/lib/npyio.py

```diff
@@ -26,11 +26,7 @@
     pickle, contextlib_nullcontext
     )
 
-if sys.version_info[0] >= 3:
-    from collections.abc import Mapping
-else:
-    from future_builtins import map
-    from collections import Mapping
+from collections.abc import Mapping
```

Comment:
group this line near L9 (rather than isolated down here)

Reply:
Done